src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        storage.put(transaction.getFruit(), transaction.getQuantity());
+    }
```

Comment:
According to the checklist, your classes should adhere to the SOLID principles. The `BalanceOperation` class should be in a `strategy` package to group it with other classes that handle different operations, as it is part of implementing the Strategy pattern.